Jenkinsfile.content

```diff
@@ -1,6 +1,12 @@
 node('vetsgov-general-purpose') {
+  if (params.ref.startsWith('vets-website')) {
+    branchName = "refs/tags/${params.ref}"
+  } else {
+    branchName = params.ref
+  }
+
```

Comment:
Am I correct that the possible values for `ref` are:

- `abcd1234....` (a github hash)
- `vets-website/v0.0.1` (a release tag)

What about separating these into two separate input params?

Reply:
Or maybe a function just to help with readability & abstraction, e.g.

```groovy
def isReleaseBuild() {
  return params.ref.startsWith('vets-website')
}
```

Reply:
Yeah thats a good question. This job is pretty much only used from here https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/Jenkinsfiles/deploys/warn-release-deploy#L107 . That build script is used in a few places so I tried to make the `prerelease` stage somewhat generic. So the question is do we keep adding specific params to a supposedly generic interface. Its kind of crappy situation and I can't help but make it not generic with this other PR https://github.com/department-of-veterans-affairs/devops/pull/4290